proto/ojo/oracle/v1/oracle.proto

```diff
@@ -41,6 +41,11 @@ message Params {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable)   = false
   ];
+  repeated Denom mandatory_list = 9 [
+    (gogoproto.moretags)     = "yaml:\"mandatory_list\"",
```

Comment:
Totally agree, everyone keeps referring to the existing code and adding yaml tags, we should remove the existing ones to avoid confusion. Feel free to make an issue if you have something else higher priority.